### ISSUE TITLE
This appeared in AS Giraffe (2022.3.1.1)

### ISSUE THREAD

**Issue by RoarGronmo** [flutter/flutter-intellij#6573]
Please file this bug report at https://github.com/flutter/flutter-intellij/issues/new.

---

## What happened

(please describe what you were doing when this exception occurred)

## Version information

Android Studio `Giraffe | 2022.3.1 Canary 1` • Flutter plugin `io.flutter 71.3.6` • Dart plugin `223.4884.65`

No Flutter sdk configured.

## Exception

null

```
com.intellij.diagnostic.PluginException: Cannot create extension (class=io.flutter.FlutterStudioInitializer) [Plugin: io.flutter]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:1058)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:70)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:31)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.java:473)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:421)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.computeExtensionList(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:243)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:39)
	at com.android.tools.idea.AndroidInitialConfigurator.activateAndroidStudioInitializerExtensions(AndroidInitialConfigurator.java:52)
	at com.android.tools.idea.AndroidInitialConfigurator.<init>(AndroidInitialConfigurator.java:30)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:49)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:1003)
	at com.intellij.serviceContainer.MyComponentAdapter.doCreateInstance(MyComponentAdapter.kt:36)
	at com.intellij.serviceContainer.BaseComponentAdapter.doCreateInstance(BaseComponentAdapter.kt:145)
	at com.intellij.serviceContainer.BaseComponentAdapter.createInstance$lambda-1(BaseComponentAdapter.kt:124)
	at com.intellij.openapi.progress.Cancellation.computeInNonCancelableSection(Cancellation.java:99)
	at com.intellij.serviceContainer.BaseComponentAdapter.createInstance(BaseComponentAdapter.kt:123)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:92)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:77)
	at com.intellij.serviceContainer.ComponentManagerImpl$createInitOldComponentsTask$1.invoke(ComponentManagerImpl.kt:401)
	at com.intellij.serviceContainer.ComponentManagerImpl$createInitOldComponentsTask$1.invoke(ComponentManagerImpl.kt:399)
	at com.intellij.idea.ApplicationLoader$initApplicationImpl$appInitializedListeners$1$1$2$1.invokeSuspend(ApplicationLoader.kt:145)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:398)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: io.flutter.FlutterStudioInitializer PluginClassLoader(plugin=PluginDescriptor(name=Flutter, id=io.flutter, descriptorPath=plugin.xml, path=~\AppData\Roaming\Google\AndroidStudioPreview2022.3\plugins\flutter-intellij, version=71.3.6, package=null, isBundled=false), packagePrefix=null, instanceId=128, state=active)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1526)
	at com.intellij.serviceContainer.ComponentManagerImplKt.access$doLoadClass(ComponentManagerImpl.kt:1)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:948)
	at com.intellij.openapi.extensions.impl.InterfaceExtensionImplementationClassResolver.resolveImplementationClass(InterfaceExtensionImplementationClassResolver.kt:24)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:49)
	... 41 more
```

**Comment by stevemessick** [comment]
This is why we do not officially support canary builds. We can't prevent the IntelliJ plugin from being loaded into Android Studio, but until the release of the first canary build, we can't build a version that would work. I'm looking into what it takes to build a version for Giraffe. With luck, it will be available in the next release (Feb 1). No promises, though.

**Comment by Nikzed** [comment]
> This is why we do not officially support canary builds. We can't prevent the IntelliJ plugin from being loaded into Android Studio, but until the release of the first canary build, we can't build a version that would work. I'm looking into what it takes to build a version for Giraffe. With luck, it will be available in the next release (Feb 1). No promises, though.

can you be more clear please how to solve this problem? Install another version of Android Studio or what

**Comment by stevemessick** [comment]
Until we can build a version of the plugin that works in AS canary, please use beta or stable.

**Comment by Nikzed** [comment]
> Until we can build a version of the plugin that works in AS canary, please use beta or stable.

I'm having problem on stable version and you closed my question with quote of this issue, thank you

**Comment by stevemessick** [comment]
This is currently blocked on
- https://github.com/JetBrains/gradle-intellij-plugin/issues/1288
- https://github.com/JetBrains/gradle-intellij-plugin/issues/1246

**Comment by stevemessick** [comment]
This will be fixed in v72.0.

**Comment by doandat943** [comment]
i had same problem
